rq/registry.py

```diff
@@ -286,11 +286,15 @@ def remove_jobs(self, timestamp=None, pipeline=None):
         score = timestamp if timestamp is not None else current_timestamp()
         return connection.zremrangebyscore(self.key, 0, score)
 
-    def get_jobs_to_schedule(self, timestamp=None):
+    def get_jobs_to_schedule(self, timestamp=None, chunk_size=None):
```

Comment:
Let’s just set `chunk_size` to default to 1000 so we don’t have to deal with two different code paths.

Reply:
LGTM, thanks